### DISCUSSION TITLE
asyncSearch по нескольким полям

### DISCUSSION THREAD

**Discussion by crick-ru** [moonshine-software/moonshine#861] [Ideas]
У полей-отношений (BelongsToMany и др.) есть метод asyncSearch(). В качестве первого параметра он принимает поле, по которому нужно производить поиск. Было бы здорово, если бы метод принимал не строку а массив. И поиск мог бы производиться по нескольким полям в связанной таблице.

Пример практического применения. База товаров. Отдельная связанная таблица с брендами производителей. В таблице могут быть русские и английские транскрипции наименований. Например, Samsung и Самсунг. Во всех возможных формах и списках логичнее выводить Samsung. А при выборе значения в поле искать проще и быстрее на русском. Не лишаясь при этом возможности искать и на английском.

Это только один самый очевидный пример. Могут быть и другие кейсы, где это было бы полезно.

**Comment by lee-to** [comment]
Не уверен что такой сахар сейчас нужен в коробке, есть ведь параметр asyncQueryCallback с помощью которого можно изменить поиск
